PillarChap/Pillar.pillar

```diff
@@ -663,8 +909,51 @@ template configuration parameter (see *@sec:confParam:template*).
 
 In this section we show how to use the ==pillar== command-line interface.
 
+One of the basic use of the command line is:
+
 [[[language=bash
-$ ./pillar export --to=latex PharoSound.pier > PharoSound.tex
+$ ./pillar export --to=latex PharoSound.pier
+# or in Windows
+$ Pharo.exe Pillar.image pillar export --to=latex
+]]]
+
+You can define an export type with the parameter ==to==. The possible exports are:
+[[[eval=true
+  PRDocumentWriter allSubclasses do: [ :subclass | subclass isAbstract ifFalse: [ stream nextPut: $-; nextPutAll: subclass writerName; lf ] ]
+]]]
+
+In the case of a ''LaTeX'' or ''Beamer'' export Pillar will generate a script (''pillarPostExport.sh'') in order to launch the compilation of all the Tex files. During the compilation the script will create a symbolic link to the output directory root named ""root"" into the directory that contains the Tex file.
+You can use this symbolic link for relatives path.
+You can customize the LaTeX command with the ==latexCommand== parameter of the configuration.
+For example for ""Enterprise Pharo"" we use:
+
+[[[language=ston
+  "latexCommand" : "lualatex --file-line-error --interaction=batchmode {fileName} 2>&1 1>/dev/null
+ret=$?
+if [[ $ret -ne 0 ]]; then
+  cat {fileNameWithoutExtension}.log
+  echo \"Can't generate the PDF!\"
+  exit 1
+fi"
+]]]
+
+For now the script is only generated for an Unix shell. Windows users must have ''MinGW'' or ''Cygwin'' in order to launch it.
+
+!!! Multiple Exports
+
+You also have the possibility to launch multiple export with the ==defaultExporters== parameter.
+For example you can add this to the ''pillar.conf'':
+
+[[[
+    "defaultExporters" : [ "html", "latex", "text" ]
+]]]
+
+Then launch:
+
+[[[language=bash
+$ ./pillar export
+# or in Windows
+$ Pharo.exe Pillar.image pillar export
 ]]]
```

Comment:
explain in 1 sentence what is the expected output when using this ==defaultExporters==